README.md

```diff
@@ -10,11 +10,11 @@ Usage
 
 This library is currently available for Scala binary versions 2.10 and 2.11.
 
-To use the latest version, include one of the following in your `build.sbt`:
+To use the latest version, include following dependency in your `build.sbt`:
```

Comment:
This should be "the following dependency"

Reply:
@larsrh updated and rebased.